past-issue/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="fragment_section_details" type="id" />
+    <item name="item_menu_activity_main_latest_issue" type="id" />
+</resources>
```

Comment:
What is this for?

Reply:
findNavController().navigate(R.id.item_menu_activity_main_latest_issue, args)